tools/helpers.py

```diff
@@ -16,3 +17,8 @@ def get_remote_data(url):
     req.add_header('Accept', 'application/json')
     json_response = urlopen(req, timeout=240).read()
     return json_response
+
+
+def url_with_params(url, params):
+    """Builds a full URL with encoded parameters"""
+    return url + "?" + urlencode(params)
```

Comment:
I was about to say single quote, but we are not consistent, in the code. we should fix that at a point

Just for your information. Another way to do it. 
but yours is working too.

```python
>>> from urllib.parse import urlunparse
>>> from urllib.parse import urlencode
>>> query_params = { "distribution": "A",
...   "type": "B",
...   "start": "C",
...   "end": "D"}
>>> query = urlencode(query_params)
>>> urlunparse(('https', 'arewehotfixingthewebyet.com', 'data.json', '', query, ''))
'https://arewehotfixingthewebyet.com/data.json?distribution=A&type=B&start=C&end=D'
```

Reply:
I went with double quotes because there is no usage of single quotes inside `api/views.py`.

Thanks for showing me `urlunparse`, that's cool! I'll absolutely use that next time if I need variable protocols, hosts, or paths!